### ISSUE TITLE
Errors with generating the namelists for CMIP esm experiments

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#237]
Running the  `generate-ec-earth-namelists-for-all-MIP-experiments.py` script gives errors for the CMIP esm experiments when comming to the point of running `drq2ppt.py` :
`````
./generate-ec-earth-namelists.sh CMIP esm-hist 1 1
./generate-ec-earth-namelists.sh CMIP esm-piControl 1 1
`````
It however appears that it goes wrong already at the data requests:
`````
  drq -m CMIP -t 1 -p 1 -e esm-hist --xls --xlsDir cmip6-data-request-m=CMIP-e=esm-hist-t=1-p=1

  drq -m CMIP -t 1 -p 1 -e esm-piControl --xls --xlsDir cmip6-data-request-m=CMIP-e=esm-piControl-t=1-p=1
`````
Maybe related with #199. 

However, this issue has not the highest priority at this moment.

**Comment by treerink** [comment]
This type of `esm` experiments need the additional flag `--esm`, so this runs in version 01.0029:
`````
drq -m CMIP -t 1 -p 1 -e esm-hist      --esm --xls --xlsDir cmip6-data-request-m=CMIP-e=esm-hist-t=1-p=1
drq -m CMIP -t 1 -p 1 -e esm-piControl --esm --xls --xlsDir cmip6-data-request-m=CMIP-e=esm-piControl-t=1-p=1
`````

**Comment by treerink** [comment]
Issue solved in 6449cbb. The following additional control output sub directories are created with `genecec`:
`````
cmip6-output-control-files/CMIP/EC-EARTH-AerChem/cmip6-experiment-CMIP-esm-hist
cmip6-output-control-files/CMIP/EC-EARTH-AerChem/cmip6-experiment-CMIP-esm-piControl
cmip6-output-control-files/CMIP/EC-EARTH-AOGCM/cmip6-experiment-CMIP-esm-hist
cmip6-output-control-files/CMIP/EC-EARTH-AOGCM/cmip6-experiment-CMIP-esm-piControl
cmip6-output-control-files/CMIP/EC-EARTH-CC/cmip6-experiment-CMIP-esm-hist
cmip6-output-control-files/CMIP/EC-EARTH-CC/cmip6-experiment-CMIP-esm-piControl
cmip6-output-control-files/CMIP/EC-EARTH-GrisIS/cmip6-experiment-CMIP-esm-hist
cmip6-output-control-files/CMIP/EC-EARTH-GrisIS/cmip6-experiment-CMIP-esm-piControl
cmip6-output-control-files/CMIP/EC-EARTH-HR/cmip6-experiment-CMIP-esm-hist
cmip6-output-control-files/CMIP/EC-EARTH-HR/cmip6-experiment-CMIP-esm-piControl
cmip6-output-control-files/CMIP/EC-EARTH-LR/cmip6-experiment-CMIP-esm-hist
cmip6-output-control-files/CMIP/EC-EARTH-LR/cmip6-experiment-CMIP-esm-piControl
cmip6-output-control-files/CMIP/EC-EARTH-Veg/cmip6-experiment-CMIP-esm-hist
cmip6-output-control-files/CMIP/EC-EARTH-Veg/cmip6-experiment-CMIP-esm-piControl
cmip6-output-control-files/CMIP/EC-EARTH-Veg-LR/cmip6-experiment-CMIP-esm-hist
cmip6-output-control-files/CMIP/EC-EARTH-Veg-LR/cmip6-experiment-CMIP-esm-piControl
`````

**Comment by treerink** [comment]
Closing.
